### ISSUE TITLE
What is the status of the repo?

### ISSUE THREAD

**Issue by Tochemey** [robfig/cron#493]
Hello @robfig. 
Nice job for putting this awesome library. I would like to know the status of the repo because it seems that some cool PRs have been provided, however no action or feedback have been taken on them.
Knowing the status of the repo can help us a lot.

Thanks.

**Comment by arpanetus** [comment]
@Tochemey seems like @SamPeng87's [fork](https://github.com/SamPeng87/cron/) is kinda active. But it needs some polishing, like some parts need to be removed, or documentation is lacking.

**Comment by janrnc** [comment]
There's a [fork](https://github.com/gdgvda/cron) I'm working on with @gdgvda. We are already looking at some cool PRs from here as a starting point. For anyone interested, we would love to consider your proposals.

**Comment by bronger** [comment]
It would be good to start a process to transfer maintainership @robfig so that the work in the PRs could be appreciated.

**Comment by elbert-chan** [comment]
> Hello @robfig. Nice job for putting this awesome library. I would like to know the status of the repo because it seems that some cool PRs have been provided, however no action or feedback have been taken on them. Knowing the status of the repo can help us a lot.
> 
> Thanks.

@robfig hello ?

**Comment by alaingilbert** [comment]
@Tochemey I made a fork that addresses almost all issues of this repo.
https://github.com/alaingilbert/cron

**Comment by Ariestattoo** [comment]
@alaingilbert Are you going to support the fork? I don't see an issues tab for reporting?

**Comment by alaingilbert** [comment]
@Ariestattoo yeah, it is supported. I enabled the "issues", I was not even aware that was something I had to "enable".

**Comment by Ariestattoo** [comment]
Thanks very much @alaingilbert. Appreciate the work you've done

**Comment by bronger** [comment]
I also thank @alaingilbert for the work.  I don’t understand, though: “This branch is 514 commits ahead of, 52 commits behind robfig/cron:master.” Have you rebased the latest 52 commits? You surely forked from _current_ master, didn’t you?

**Comment by alaingilbert** [comment]
I started from a fork that I had a very long time ago.
Then I manually took the newer parser/spec.
Otherwise whatever was in cron.go has almost been rewritten entirely.
